src/librustc_unicode/u_str.rs

```diff
@@ -27,9 +27,16 @@ use core::str::Split;
 
 use tables::grapheme::GraphemeCat;
 
-/// An iterator over the words of a string, separated by a sequence of whitespace
+#[deprecated(reason = "struct Words is being replaced by struct SplitWhitespace",
+             since = "1.0.0")]
+#[unstable(feature = "unicode",
```

Comment:
Generally, no, but the stabilization of `Words` here I believe was an accident, so I think it's fine to go back to `unstabe`.